daceml/onnx/nodes/onnx_op.py

```diff
@@ -52,12 +94,16 @@ class ONNXOp(nd.LibraryNode):
     def iter_outputs_in_onnx_order(self, state):
```

Comment:
would be nice to have type hints here and below. There's also a return value that's missing from the doc.